packages/core/src/transitions/useTransition.ts

```diff
@@ -1,14 +1,17 @@
+import {SignalValue} from '../signals/types';
 import {useScene} from '../utils';
 
 /**
  * Transition to the current scene by altering the Context2D before scenes are rendered.
  *
  * @param current - The callback to use before the current scene is rendered.
  * @param previous - The callback to use before the previous scene is rendered.
+ * @param previousOnTop - A boolean or signal indicating whether the previous scene should be rendered on top of the current scene.
```

Comment:
Lines should not exceed 80 characters

Reply:
Fixed, didn't realise as the linter did not complain. Would it be an idea to add a `max-len` rule?